lib/tasks/populate_canonical_models.rake

```diff
@@ -2,57 +2,80 @@
 
 require 'json'
 
+FALSEY_VALUES = [false, 'false'].freeze
```

Comment:
rubocop-performance indicates that including array literals in a loop unnecessarily slows down execution. Since this array occurs in multiple tasks, it made more sense to declare the constant outside the tasks so I didn't have to declare it more than once. Ordinarily, arguments passed into Rake tasks are typecast to booleans, but in case the command line argument `false` is interpreted as a string, I'm checking for the string value `'false'` as well.